README.md

```diff
@@ -11,10 +11,16 @@ AGRC manages the additions, replacements, and deletions of SGID items in the bro
 
 1. When an application, dataset, or both needs to be deleted or moved to a different long-term storage, [create a new issue](https://github.com/agrc/cemetery/issues/new/choose)
 1. Choose the `Deprecate SGID Dataset or Web Application` template from the buttons
-1. Tag the three triagers (one from each team) in the triage section
-1. The triagers will choose which items need to be handled and tag the appropriate people to handle them
-   - Items that don't need to be handled should be struck through (wrap the line with `~`'s) instead of being deleted
-1. The people assigned to different issues will check the boxes as they are completed and comment on the issue to notify the rest of the group.
+1. Tag the three triagers (one from each team) in the triage section:
+   - [ ] Data Team Triage (@gregbunce)
+1. The triagers will choose which items need to be handled and tag the appropriate people to handle them:
+   - [ ] ArcGIS Online (@jacobdadams)
+   - Items that don't need to be handled should be struck through (wrap the line with `~`'s) instead of being deleted:
+      - `~ArcGISOnline (assigned to)~` (note there are no spaces between the `~`'s and the text to be struck out)
+   - Once the triager has identified any steps required for their team or made any relevant comments, check the box to indicate their triage is complete:
```

Comment:
**suggestion**: Make it more apparent who should be the one that checks the box.

```suggestion
   - Once the triager has identified any steps required for their team or made any relevant comments, they should check the box to indicate that their triage is complete:
```

Reply:
Good call.